### PR TITLE
[Relay] Show ‘UNABLE TO LOAD’ view on GraphQL errors.

### PR DESCRIPTION
**Pull request by alloy**
https://artsyproduct.atlassian.net/browse/IN-59 #resolve

#skip_new_tests

### CODE REVIEW

File: src/lib/metaphysics.ts [hunk 1]
```diff
@@ -27,6 +27,15 @@ export function metaphysics<T>(
       }
     })
     .then<T>(response => response.json())
+    .then(json => {
+      if (json.errors) {
+        errors.forEach(console.error)
```
Comment (alloy) [inline src/lib/metaphysics.ts]: This is going to go to Sentry, which really indicates server-side errors, but might be good to know if the app suffers from specific endpoints not working well?

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/__tests__/metaphysics-tests.ts`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  [danger-plugin-jest] Could not read test results. Danger cannot pass or fail the build.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/metaphysics.ts [hunk 2]
```diff
@@ -26,7 +26,16 @@ export function metaphysics<T>(
         throw error
       }
     })
-    .then<T>(response => response.json())
+    .then<T & { errors: any[] }>(response => response.json())
```
Comment (orta) [inline src/lib/metaphysics.ts]: nice touch with the types

File: src/lib/relay/config.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ if (Emission && Emission.gravityAPIHost && Emission.metaphysicsAPIHost) {
   metaphysicsURL = "https://metaphysics-production.artsy.net"
   gravityURL = "https://api.artsy.net"
 }
+metaphysicsURL = "http://localhost:5001"
```
Comment (orta) [inline src/lib/relay/config.tsx]: nope, I imagine

Comment (alloy) [reply src/lib/relay/config.tsx]: Thanks 🙏🏽

**Pull request merged**